packages/valory/skills/decision_maker_abci/models.py

```diff
@@ -387,11 +400,13 @@ def _ensure_chatui_store(self) -> Any:
             current_store["trading_strategy"] = trading_strategy_yaml
             current_store["initial_trading_strategy"] = trading_strategy_yaml
 
-        with open(chatui_store_path, "w") as f:
-            json.dump(current_store, f, indent=4)
+        self._set_json_store(current_store)
 
         self._chat_ui_params = ChatUIParams(**current_store)
 
+        if self._chat_ui_params is None:
+            raise ValueError("The chat UI parameters have not been set!")
+
```

Comment:
Context: https://github.com/valory-xyz/trader/pull/460#discussion_r2195508353

I noticed just now that this is not possible, right? I also noticed that you kept the check for typing reasons in the `chat_ui_params` property. Therefore, let's remove this check from here as it is unnecessary.